### ISSUE TITLE
Fix wp-stateless Body content media URL rewrite setting

### ISSUE THREAD

**Issue by jlyon** [proudcity/wp-proudcity#1038]
TLDR:
Image urls are getting set to urls like https://beta.proudcity.com/wp-content/uploads/Former_Danvers_State_Hospital_Danvers_MA-300x225.jpg instead of https://storage.googleapis.com/proudcity/${APP}/uploads.

FULL DETAILS:

The `WP_STATELESS_MEDIA_REWRITE_BODY_CONTENT_URL` ENV var that we were using to force the "Body content media URL rewrite." wp-stateless setting is no longer getting read by the version of the wp-stateless plugin that we are now using.

Most (all?) production sites already had this setting manually checked (stored in the db), however our template and other TEST sites did not get this.  This resulted in the images successfully being copied to GCE, but the urls still pointing to their local copy, which gets deleted during a deployment.

**Comment by jlyon** [comment]
Deploying as an update script in Hotfix 1.43.2
